tests/lib/token-manager-test.js

```diff
@@ -630,7 +629,181 @@ describe('token-manager', function() {
 				.then(tokens => {
 					assert.equal(tokens, tokenInfo);
 				});
-		});
+		}).timeout(5000);
+
+		it('should retry with new jti claim when server rejects because of rate limiting', function() {
+
+			var firstTokenParams = {
+				grant_type: 'urn:ietf:params:oauth:grant-type:jwt-bearer',
+				assertion: TEST_WEB_TOKEN
+			};
+
+			var newJWT = 'jhdhioeiu4yo34875twudgshdgr';
```

Comment:
How was this generated?  It should be a freshly-generated key from a non-Box account...

Reply:
This is taken from other tests in the file.